services/src/contexts/migrations/migration_0000_initial.rs

```diff
@@ -0,0 +1,105 @@
+use async_trait::async_trait;
+use tokio_postgres::Transaction;
+
+use crate::{
+    error::Result,
+    layers::{
+        add_from_directory::UNSORTED_COLLECTION_ID, storage::INTERNAL_LAYER_DB_ROOT_COLLECTION_ID,
+    },
+};
+
+use super::database_migration::{DatabaseVersion, Migration};
+
+pub struct Migration0000Initial;
+
+#[async_trait]
+impl Migration for Migration0000Initial {
+    fn prev_version(&self) -> Option<DatabaseVersion> {
+        None
+    }
```

Comment:
Why do we need this here?
What about the list of all migrations… we could have a discrepancy, there.

Reply:
yeah, but that discrepancy would be caught. If we do not specify the previous version, we could forget one in the list of all migrations and it wouldn't get noticed.